.github/workflows/bmad-qa-gate-fast.yml

```
@@ -0,0 +1,71 @@
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  qa-gate:
+    # Use self-hosted runner if available, fallback to GitHub-hosted
+    runs-on: ${{ github.repository == 'auldsyababua/bigsirflrts' && github.actor == 'auldsyababua' && 'self-hosted' || 'ubuntu-latest' }}
+    timeout-minutes: 15
+    
+    # Continue on GitHub-hosted if self-hosted is unavailable
+    continue-on-error: ${{ matrix.runner == 'self-hosted' }}
```

actionlint (1.7.7)

property "runner" is not defined in object type {}

(expression)